src/lsp/cobol_lsp/lsp_server.ml

```diff
@@ -82,6 +82,7 @@ module TYPES = struct
   and exit_status = (unit, string) result
 
   type 'a error =
+    | CopybookRenamingForbidden
```

Comment:
```suggestion
    | ForbiddenRenamingInCopybook
```

Reply:
Actually this is not the correct place for such a constructor. This place is for errors that relate to the protocol itself.